src/plugins/graph/models/graph-model.ts

```diff
@@ -448,6 +459,27 @@ export const GraphModel = TileContentModel
           }
         ));
       }
+
+      // Display a plotted function when this is linked to a SharedVariableModel
+      addDisposer(self, reaction(
+        () => {
+          const smm = getSharedModelManager(self);
+          let sharedVariableModels;
+          if (smm?.isReady) {
+            sharedVariableModels = smm.getTileSharedModelsByType(self, SharedVariables);
+          }
+          return sharedVariableModels;
+        },
+        (sharedVariableModels) => {
+          if (sharedVariableModels && sharedVariableModels.length > 0) {
+            const plottedFunctionAdornment = PlottedFunctionAdornmentModel.create();
+            plottedFunctionAdornment.addPlottedFunction(x => x**2);
+            self.showAdornment(plottedFunctionAdornment);
```

Comment:
When we get further along, I think we should try to store this adornment in `self.adornments`.  So when a diagram tile is linked with the graph this adornment would get created and added. And when a the "new series" (or whatever that ends up being called) a new PlottedFunctionAdornmentModel would get created and added to `self.adornments`.  When the graph is saved this adornment will get saved in the state, so then it shouldn't be necessary to have this afterAttach code anymore. 

Initially the PlottedFunctionAdornmentModel can be added directly to `AdornmentModelUnion`.  When we add the graph plugin system, we can make the `AdornmentModelUnion` dynamic, and when the shared variable model adds the variable plugin to the graph it would register the `PlottedFunctionAdornmentModel` with this dynamic union.

Reply:
`self.showAdornment` adds the adornment to `self.adornments` if it's not already there, so the `PlottedFunctionAdornment` is already getting saved in the model.

This reaction handles both showing the adornment when there's a linked `SharedVariables`, and hiding the adornment when there isn't. Are we going to have to add calls to show and hide the adornment wherever `SharedVariables` could get linked or unlinked without a reaction like this? Maybe to hide it we only render it when there's a linked `SharedVariables`?